acados/ocp_nlp/ocp_nlp_ddp.c

```diff
@@ -662,6 +650,7 @@ int ocp_nlp_ddp(void *config_, void *dims_, void *nlp_in_, void *nlp_out_,
     ocp_nlp_memory *nlp_mem = mem->nlp_mem;
     ocp_qp_xcond_solver_config *qp_solver = config->qp_solver;
     ocp_nlp_res *nlp_res = nlp_mem->nlp_res;
+    ocp_nlp_timings *timings = nlp_mem->nlp_timings;
```

Comment:
Call `nlp_timings` to make consistent with SQP?

Reply:
done